doc/neps/nep-0036-fair-play.rst

```diff
@@ -0,0 +1,180 @@
+==================
+NEP 36 — Fair Play
+==================
+
+:Author: Stéfan van der Walt <stefanv@berkeley.edu>
+:Status: Draft
+:Type: Informational
+:Created: 2019-10-24
+:Resolution: Draft
+
+
+Abstract
+--------
+
+This document describes Rules of Play for engaging with the NumPy
+project as an external entity.  It discusses usage of the name NumPy,
+modified distributions of NumPy, and how to best communicate
+with the project when patched versions are deployed.
```

Comment:
```suggestion
This document describes Rules of Play for engaging with the NumPy
project as an external entity. 
```
```suggestion
This document sets out Rules of Play for companies and outside developers that engage with the NumPy project. It covers:
- Restrictions on use of the NumPy name
- How and whether to publish a modified distribution
- How to make us aware of patched versions
```
Make it immediately clear who this is addressed to.  Make clear that this is a defining document. Make the scope explicit.